internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -231,9 +235,13 @@ func renderMountPoints(input map[string]manifest.Volume) ([]*template.MountPoint
 	output := []*template.MountPoint{}
 	for name, volume := range input {
 		// ContainerPath must be specified.
-		if volume.ContainerPath == nil {
+		if aws.StringValue(volume.ContainerPath) == "" {
 			return nil, errVolNoContainerPath
 		}
+		path := aws.StringValue(volume.ContainerPath)
```

Comment:
`renderMountPoints` and `renderSidecarMountPoints` look pretty similar, I wonder if there is a way for us to reuse the logic?

Reply:
Good idea. Addressed in my latest push.